### ISSUE TITLE
Restore Mysql dump - Scenario 2

### ISSUE THREAD

**Issue by jnak** [engineyard/ey-docs#279]
Hi,

A quick heads up to let you know there is little bug in the doc regarding Mysql dump loading at http://docs.engineyard.com/database-restore.html

It should be 
gunzip < [filename] | mysql -u [username] --password=[password] -h [database host] [app_name]

instead of
gunzip < [filename] | mysql -u [username] -p [password] -h [database host] [app_name] 

Hope this helps!
J.

**Comment by janeday** [comment]
Hi J,

Thanks very much for your message. Sorry that you had problems, but thank you for letting me know. 

Did you try without a space between the -p and the password?   For example:  -pmypassword   ?

In your message, it looks like there is a space after the -p  .

I'll work with one of the Engine Yard database experts to figure out the best way to fix that step. 

best regards, J

**Comment by jnak** [comment]
If I remember well, I tried both with and without a space.
Regards, J.

On Wed, Jan 25, 2012 at 1:27 PM, Jane
reply@reply.github.com
wrote:

**Comment by janeday** [comment]
Our security guy suggests that best practice is to use -p without the password and then get prompted for the password -- that way the database password doesn't get saved in the shell's history. So, I've changed the Engine Yard database docs to use that syntax and added a word of explanation on this page http://docs.engineyard.com/database-password.html
My database experts swear that the -pPassword should work, so I've included both syntaxes in that explanatory paragraph. 
Thanks again for raising this issue and helping to make Engine Yard docs better for all. 
Sincerely, Jane
